x/evm/types/storage.go

```diff
@@ -50,23 +51,23 @@ func (s Storage) Copy() Storage {
 
 // State represents a single Storage key value pair item.
 type State struct {
-	Key   ethcmn.Hash `json:"key"`
-	Value ethcmn.Hash `json:"value"`
+	Key   string `json:"key"`
+	Value string `json:"value"`
 }
 
 // Validate performs a basic validation of the State fields.
 func (s State) Validate() error {
-	if bytes.Equal(s.Key.Bytes(), ethcmn.Hash{}.Bytes()) {
+	if ethermint.IsEmptyHash(s.Key) {
 		return sdkerrors.Wrap(ErrInvalidState, "state key hash cannot be empty")
 	}
 	// NOTE: state value can be empty
 	return nil
 }
 
 // NewState creates a new State instance
-func NewState(key, value ethcmn.Hash) State {
+func NewState(key, value ethcmn.Hash) State { // nolint: interfacer
```

Comment:
what is this nolint?

Reply:
otherwise the linter expects the `fmt.Stringer` interface, which is not what we want